### ISSUE TITLE
KeyError: 'cpus' in version_table() function

### ISSUE THREAD

**Issue by mmmm1998** [qutip/qutip#1004]
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from qutip.ipynbtools import version_table
>>> version_table()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mmmm1998/.local/lib/python3.6/site-packages/qutip/ipynbtools.py", line 103, in version_table
    ("Number of CPUs", qutip.hardware_info.hardware_info()['cpus']),
KeyError: 'cpus'
```

I installed qutip version 4.3.1 via pip3.

**Comment by nathanshammah** [comment]
I cannot reproduce the error as I am on darwin. 
It might be something about `_linux_hardware_info`.

**Comment by mmmm1998** [comment]
Thanks for the clue, it was helpfull.
The source of problem is localization of `lscpu` util, which used by `_linux_hardware_info()` function.
The function expects `'Socket(s)'`, `'Core(s) per socket'`, etc in `lscpu` output, but in my system it actually `'Потоков на ядро'`, `'Сокетов'`, etc.
I think, maybe better less system dependent ways to calculate cpu count, for example, we could use `multiprocessing` module and it's `cpu_count` function.
Or show exception/error, if we try to use lscpu and `LANG` is not english (or we couldn't found mentioned above `'Socket(s)'`).
Ideas?

**Comment by hodgestar** [comment]
Fixed in #1758.
